README.md

```diff
@@ -168,6 +170,60 @@ export ASB_NGSA_MI_RESOURCE_ID=$(az identity create -g $ASB_RG_CORE -n $ASB_NGSA
 
 ```
 
+### Create managed identity for ngsa-app
+
+```bash
+
+export ASB_NGSA_MI_NAME="${ASB_DEPLOYMENT_NAME}-ngsa-id"
+
+export ASB_NGSA_MI_RESOURCE_ID=$(az identity create -g $ASB_RG_CORE -n $ASB_NGSA_MI_NAME --query "id" -o tsv)
+
+# save env vars
+./saveenv.sh -y
+
+```
+
+### Verify Firewall Assignment
+
+Verify that the developer's I.P. address is added to the firewall for the application.  This can be done by visiting the portal navigate to the Cosmos DB account in question, and select the Networking tab.  From the Networking screen for the Cosmos DB account follow the screen to add the I.P. address.  This can be accomplished by selecting the link titled  "+ Add my current IP xxx.xxx.xxx" and follow the prompts to save the I.P. address added.  
+
+Please visit this [link](https://learn.microsoft.com/en-us/azure/cosmos-db/how-to-configure-firewall) for more detailed instructions.  
+
+### Assign User to Cosmos DB
+
+```bash
+  
+  local Assigee_Object_Id = 'Users Object Id'
+
+  az login --tenant 'XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'
+  az account set --subscription 'XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX'
+
+  # Variables
+  # built in Cosmos DB roles
+  local roles=("00000000-0000-0000-0000-000000000001" "00000000-0000-0000-0000-000000000002") #Array of built in Cosmos roles
```

Comment:
If you have 00000000-0000-0000-0000-000000000002, you should not need 00000000-0000-0000-0000-000000000001. [See docs](https://learn.microsoft.com/en-us/azure/cosmos-db/how-to-setup-rbac#built-in-role-definitions)

Reply:
I have fixed this.